### ISSUE TITLE
Using the Aware Library but can't find symbol class Aware

### ISSUE THREAD

**Issue by kyleesantos** [denzilferreira/aware-client#231]
I am trying to create a standalone application in android studio that uses Aware as a built-in library. I followed the step-by-step process on the Aware framework site and it seemed to work until the last step of actually using the sensors. I do not need a plugin for my application, I just want to access the sensor data directly. I tried putting the sample code for accessing accelerometer data in my MainActivity class but it is raising errors that it cannot find the symbol class Aware or the variables Aware_preferences. Do I have to import the Aware.java code in the github in order for it to successfully build? Or does using Aware as the built-in library allow me to skip that step?

**Comment by denzilferreira** [comment]
Make sure your project has downloaded the library to your project. If it can't find the classes, it's because the library is not yet on your project.

**Comment by kmarkiv** [comment]
Hi @denzilferreira I seem to have the same issue. I have added the project on gradle/manually but it says package com.aware not found. I'm able to import other libraries though. 

![image](https://user-images.githubusercontent.com/933844/48244619-a8c68100-e3b4-11e8-8492-5da7aa4bed7c.png)

**Comment by denzilferreira** [comment]
Make sure to be targeting SDK 28 on your app.

**Comment by kmarkiv** [comment]
HI @denzilferreira, 

I created a sample project in API 28.  Added aware and a few dependencies mentioned here. 

http://www.awareframework.com/creating-a-standalone-application/

![image](https://user-images.githubusercontent.com/933844/48293715-70738100-e44e-11e8-9985-38fde6d2e99d.png)

Let me know if there is an updated tutorial on this and I'll try to follow it. I am not a regular Android developer so I might be missing something. Thanks in advance.
